lib/cbus_elixir/app/meetings.ex

```diff
@@ -5,12 +5,48 @@ defmodule CbusElixir.App.Meetings do
 
   import Ecto.Query
 
+  alias Ecto
   alias CbusElixir.Repo
   alias CbusElixir.App.Meeting
+  alias CbusElixir.Pagination
+  alias CbusElixir.App.Speaker
+  alias CbusElixir.App.Attendee
+
+  @doc """
+  Fetch past meetings
+  """
+
+  @spec meetings_for_page_query() :: Ecto.Query.t()
+  def meetings_for_page_query() do
+    from(m in Meeting,
+      left_join: s in assoc(m, :speakers),
+      where: m.date < ^DateTime.utc_now(),
+      order_by: [desc: m.date],
+      distinct: m.date
```

Comment:
Oh I see. So you're able to get past the limit result issue by calling distinct?